pnpbridge/src/adapters/src/impinj_reader_r700/support_files/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../../pnpbridge/src/pnpbridge_config_schema.json",
+  "pnp_bridge_connection_parameters": {
+    "connection_type" : "dps",
+    "root_interface_model_id": "dtmi:impinj:R700;1",
+    "auth_parameters": {
+        "auth_type": "symmetric_key",
+        "symmetric_key": "lZKGVYbHYRcaJkeJt4COQ6RxtBb61IHmO783Gx13lks="
+    },
+    "dps_parameters": {
+        "global_prov_uri": "global.azure-devices-provisioning.net",
+        "id_scope": "0ne001C686E",
+        "device_id": "kwenner-R700"
+    }
+},
+  "pnp_bridge_debug_trace": false,
+  "pnp_bridge_config_source": "local",
+  "_comment_devices": "Array of devices for Azure Pnp interface should be published",
+  "pnp_bridge_interface_components": [
+      {
+          "_comment": "Impinj Reader Device (R700)",
+          "pnp_bridge_component_name": "R700",
+          "pnp_bridge_adapter_id": "impinj-reader-r700",
+          "pnp_bridge_adapter_config": {
+              "hostname": "192.168.1.14",
+              "username": "root",
+              "password": "impinj"
```

Comment:
Check-in place holders instead of actual values here that may be sensitive.

Reply:
resolved before PR #52 merged